docs/tables/mongodbatlas_custom_db_role.md

```diff
@@ -0,0 +1,40 @@
+# Table: mongodbatlas_custom_db_role
+
+Custom roles supports a subset of MongoDB privilege actions. These are defined at the project level, for all clusters in the project.
+
+Using custom MongoDB enables you to specify custom sets of actions which cannot be described by the built-in Atlas database user privileges.
+
+## Examples
+
+### Basic Info
+
+```sql
+select
+  role_name,
+  actions
+from
+  mongodbatlas_custom_db_role
+```
+
+### List roles which have the 'FIND' action defined
+
+```sql
+select
+  *
+from
+  mongodbatlas_custom_db_role t,
+  jsonb_array_elements(t.actions) j
+where
+  j ->> 'action' = 'FIND'
+```
+
+### List roles which have at least one inhertied role
+
+```sql
+select
+  *
```

Comment:
We should avoid using `select * ...` in anything other than the basic queries. We should be deliberate with what columns our example queries return, i.e., relevant columns to the query's intent, especially since Steampipe only calls hydrate functions when necessary